Controller/DbControllers/RequestController.cs

```diff
@@ -40,37 +40,49 @@ public virtual List<Request> GetArtistRequests()
             return GetFilteredList(r => r.RequestType == RequestType.Artist);
         }
 
-        /**
-         * Approves a request from a user to become an artist
-         *
-         * @param requestID The ID of the request in question
-         */
-        public void ApproveUser(int requestID)
+        public virtual RequestArtistResults RequestArtist(string ArtistName, string ArtistReason)
```

Comment:
Hier moet nog functie documentatie boven

Reply:
is aangepast